src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -318,7 +318,8 @@
             :params     [(js/JSON.stringify session-info)]
             :on-success (fn []
                           (log/info "Wallet Connect session persisted")
-                          (rf/dispatch [:wallet-connect/fetch-persisted-sessions]))
+                          (rf/dispatch [:wallet-connect/fetch-persisted-sessions])
+                          (rf/dispatch [:activity-center.notifications/fetch-unread-count]))
```

Comment:
The `fetch-persisted-sessions` can be called multiple times in the lifecycle of the app: ex: while switching networks. 

Do we need to called `fetch-unread-count` multiple times or just once during init?

Reply:
yes, there's a new notification for each added session